tools/Config.mk

```diff
@@ -392,8 +420,11 @@ endef
 #   CONFIG_WINDOWS_NATIVE - Defined for a Windows native build
 
 define ARCHIVE_ADD
-	@echo "AR (add): ${shell basename $(1)} $(2)"
+	$(if $(BASH),$(eval SHELL=$(BASH)))
+	$(ECHO_BEGIN)"AR (add): ${shell basename $(1)} $(2) "
```

Comment:
could we move if into ECHO_BEGIN

Reply:
emm... ECHO_BEGIN is a variable, and according to the syntax of the makefile, @echo must be on the line first...

Reply:
Done ,I set SHELL=BASH as default